vro-streamlit/src/vro_streamlit/auth/auth_service.py

```diff
@@ -0,0 +1,9 @@
+from vro_streamlit.auth.user import User
+
+
+def log_in() -> User:
+    return User('test')
+
+
+def log_out() -> bool:
+    return True
```

Comment:
I would maybe add TODOs here:
```
# auth_service.py
def log_in() -> User:
    # TODO: Implement real authentication logic
    pass

def log_out() -> bool:
    # TODO: Implement real logout logic
    pass
```

Reply:
I would! But I'm almost ready to push another PR that does just this! 😬